src/helpers/utils.ts

```diff
@@ -103,23 +136,41 @@ export function runCommand(cmd: string, requiredRepoType = RepoType.Undefined) {
   } else if (getRepoType() !== requiredRepoType) {
     vscode.window.showErrorMessage(`Command works only with ${requiredRepoType} projects!`);
   } else {
-    const project = resolve(workspace.uri.fsPath, 'package.json');
-
-    try {
-      const { scripts = {} } = __non_webpack_require__(project) || {};
-      const candidates = Object.keys(scripts).filter((m) => scripts[m].trim().startsWith(cmd));
-      const shellCommand =
-        candidates.length === 0 ? cmd : candidates.length === 1 ? scripts[candidates.pop() ?? ''] : undefined;
-
-      if (shellCommand !== undefined) {
-        execCommand(shellCommand);
-      } else {
-        vscode.window.showQuickPick(candidates).then(execCommand);
+    const nodeModulesAreAvailable = existsSync(`${workspace.uri.fsPath}/node_modules`);
```

Comment:
This is not enough. As mentioned we need to check if `node_modules/.bin/piral` exists. If it does not the dependencies are not installed correctly and should be installed (again).